lib/samples/query_table_statistics/query_table_statistics_sample.dart

```diff
@@ -29,117 +29,190 @@ class QueryTableStatisticsSample extends StatefulWidget {
-      );
-    }
-
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
-    final featureLayer = FeatureLayer.withFeatureTable(_serviceFeatureTable);
-    map.operationalLayers.add(featureLayer);
-
-    _mapViewController.arcGISMap = map;
-  }
+  // A flag to display the query settings.
+  var _toggleQuerySettings = true;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Stack(
-        alignment: Alignment.center,
-        children: [
-          ArcGISMapView(
-            controllerProvider: () => _mapViewController,
-          ),
-          Positioned(
-            width: 350,
-            height: 180,
-            bottom: 60,
-            child: DecoratedBox(
-              decoration: BoxDecoration(
-                color: Colors.white.withOpacity(0.8),
-              ),
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  SwitchListTile(
-                    title: const Text('Only cities in current extent'),
-                    value: _onlyCitiesInCurrentExtent,
-                    onChanged: (value) {
-                      setState(() => _onlyCitiesInCurrentExtent = value);
-                    },
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
                   ),
-                  SwitchListTile(
-                    title: const Text('Only cities greater than 5M'),
-                    value: _onlyCitiesGreaterThan5M,
-                    onChanged: (value) {
-                      setState(() => _onlyCitiesGreaterThan5M = value);
-                    },
-                  ),
-                  TextButton(
-                    onPressed: queryStatistics,
-                    child: const Text(
-                      'Get statistics',
-                    ),
-                  ),
-                ],
+                ),
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: SizedBox.expand(
+                child: Container(
+                  color: Colors.white30,
+                  child: const Center(child: CircularProgressIndicator()),
+                ),
               ),
             ),
-          ),
-        ],
+
+            Visibility(
+              visible: _toggleQuerySettings,
+              child: querySettings(context),
+            ),
+          ],
+        ),
       ),
+      bottomSheet: null, //querySettings(context),
```

Comment:
What would you think about using the bottomSheet idea, like "Densify and generalize geometry" and "Create geodetic and planar buffers"? That'll give the samples more consistency and it has a "Close" button.